commit 16f00548516532c286771834b7e4910cda30f0a9
Author: vitaliy-fuks <vitaliy.fuks@kontagent.com>
Date:   Thu Jun 7 13:27:55 2012 +0000

    Swapped order of parameters in "Client %r passed an invalid value for timestamp %r" error message.

diff --git a/sentry/coreapi.py b/sentry/coreapi.py
index 0b694b1a5f..c488ebbfe2 100644
--- a/sentry/coreapi.py
+++ b/sentry/coreapi.py
@@ -291,8 +291,8 @@ def validate_data(project, data, client=None):
         except InvalidTimestamp:
             # Log the error, remove the timestamp, and continue
             logger.error('Client %r passed an invalid value for timestamp %r' % (
-                data['timestamp'],
                 client or '<unknown client>',
+                data['timestamp'],
             ))
             del data['timestamp']
 
